PhysicsTools/PatAlgos/python/slimming/puppiForMET_cff.py

```diff
@@ -21,8 +21,8 @@ def makePuppies( process ):
     process.puppiMerged = cms.EDProducer("CandViewMerger",src = cms.VInputTag( 'puppiNoLep','pfLeptonsPUPPET'))
     process.load('CommonTools.PileupAlgos.PhotonPuppi_cff')
     process.puppiForMET = process.puppiPhoton.clone()
-    process.puppiForMET.puppiCandName    = 'puppiMerged'
-
+    #process.puppiForMET.puppiCandName    = 'puppiMerged'
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant